RetailCoder.VBE/Root/RubberduckModule.cs

```diff
@@ -425,6 +429,10 @@ private IEnumerable<ICommandMenuItem> GetRubberduckCommandBarItems()
 
         private IEnumerable<IMenuItem> GetRubberduckMenuItems()
         {
+            //This bind needs to occur before the the new array is built by the function calls
```

Comment:
Thanks for the feedback on this particular change.  Using Rebind did not succeed...I'm thinking it is because the Inspection objects are constructed just once (when the menu items are created) and there is never another opportunity to inject the redefined dependency with Rebind(). (True?)  I've moved the Bind definition to a sequentially (and contextually) correct place in the body of the "Load()' function.